src/dags/importscripts/import_financien_grootboek.py

```diff
@@ -0,0 +1,138 @@
+    dwh_ora_engine = get_ora_engine("oracle_dwh_ami")
+    date_fmt = "%Y%d%m"
+    with dwh_ora_engine.connect() as connection:
+        df = pd.read_sql(
+            """
+            SELECT
+             gb.DATUM_BOEKING as "datum_boeking"
+            ,gb.BEDRAG_BOEKING as "bedrag_boeking"
+            ,gb.OMSCHRIJVING as "boeking_omschrijving"
+            ,gb.OPMERKING as "boeking_opmerking"
+            ,gbr.HOOFDREKENING_CODE as "hoofdrekening_code"
+            ,gbr.REKENING_OMSCHRIJVING as "rekening_omschrijving"
+            ,gbr.BEDRIJFSEENHEID_CODE as "bedrijfseenheid_code"
+            ,gbr.BEDRIJFSEENHEID_OMSCHRIJVING as "bedrijfseenheid_omschrijving"
+            ,gb.SUBGROOTBOEK as "subgrootboek"
+            ,gb.SUBGROOTBOEKTYPE as "subgrootboektype"
+            ,vpr.VPL_VERPLICHTINGNUMMER as "vpl_verplichtingnummer"
+            ,vpr.VPL_OMSCHRIJVING as "vpl_omschrijving"
+            ,vpr.VPL_JAAR as "vpl_jaar"
+            ,vpr.VPL_INKOOPCONTRACTCODE as "vpl_inkoopcontractcode"
+            ,vpr.VPL_STATUS as "vpl_status"
+            ,vpr.VPR_DTM_LSTE_AFBOEKING as "vpr_datum_laatste_afboeking"
+            ,vpr.VPR_BEDRAG_OORSPRONKELIJK as "vpr_bedrag_oorspronkelijk"
+            ,vpr.VPR_BEDRAG_AFGEBOEKT as "vpr_bedrag_afgeboekt"
+            ,vpr.VPR_BEDRAG_RESTANT as "vpr_bedrag_restant"
+            ,vpr.VPR_REGELNUMMER as "vpr_regelnummer"
+            ,vpr.VPR_OMSCHRIJVING as "vpr_omschrijving"
+            ,vpr.VPR_OPMERKING as "vpr_opmerking"
+            ,vpr.VPR_DATUM_GROOTBOEK as "vpr_datum_grootboek"
+            ,vpr.VPR_DATUMVERPLICHTING as "vpr_datumverplichting"
+            ,wor.CODE as "werkorder_code"
+            ,wor.OMSCHRIJVING  as "werkorder_omschrijving"
+            ,wor.STATUS as "werkorder_status"
+            ,wor.STATUS_OPMERKING as "werkorder_status_opmerking"
+            ,rel.code as "relatie_code"
+            ,rel.naam as "relatie_naam"
+            ,rel.IND_CREDITEUR as "indicatie_crediteur"
+            ,rel.IND_DEBITEUR as "indicatie_debiteur"
+            ,org.SUBTEAM_CODE as "subteam_code"
+            ,org.SUBTEAM_NAAM as "subteam_naam"
+            ,org.TEAM_CODE as "team_code"
+            ,org.TEAM_NAAM as "team_naam"
+            ,org.AFDELING_CODE as "afdeling_code"
+            ,org.AFDELING_NAAM as "afdeling_naam"
+            ,org.RVE_CODE as "rve_code"
+            ,org.RVE_NAAM as "rve_naam"
+            ,org.CLUSTER_CODE as "cluster_code"
+            ,org.CLUSTER_NAAM as "cluster_naam"
+            FROM DATAHUB.FI2_DHB_FCT_GROOTBOEK GB
+            INNER JOIN DATAHUB.FI2_DHB_DIM_RELATIE REL ON REL.ID = GB.REL_ID
+            INNER JOIN DATAHUB.FI2_DHB_DIM_ORGANISATIE org ON ORG.ID = GB.ORG_ID
+            INNER JOIN DATAHUB.FI2_DHB_DIM_GROOTBOEKREKENING gbr ON gbr.ID = GB.gbr_ID
+            INNER JOIN DATAHUB.FI2_DHB_DIM_VERPLICHTINGREGEL vpr ON vpr.ID = GB.VPR_ID
+            INNER JOIN DATAHUB.FI2_DHB_DIM_WERKORDER wor ON wor.ID = GB.WOR_ID
+            WHERE 1=1
```

Comment:
I see this often in your SQL statements. It does nothing. Might I suggest you remove it?